README.md

```diff
@@ -1,5 +1,5 @@
 <!-- @format -->
-> # Black Lives Matter 🖤 Free Palestine 🇵🇸
+> # Black Lives Matter 🖤 Make Code Not War ♥️ 🌿 🌏
```

Comment:
Why Palestine is a war while Israel is the heaven? 😆  I don't think you're living in the world

Reply:
The change doesn't mention Israel or Palestine 🤨

Reply:
When you change something to another thing you can't detach the context. Why he thought about War then? If it was a clean change I'd definitely agree with you. Try to think about it and simulate the change, anyone can infer what I'm referring to.

Reply:
He didn't say anything about Israel heaven or Palestine war that you mentioned in your original comment.

As hundreds of other people have already mentioned it should never have entered the repo to begin with.